CODE_OF_CONDUCT.md

```diff
@@ -5,7 +5,7 @@ all, regardless of level of experience, gender, gender identity and expression,
 sexual orientation, disability, personal appearance, body size, race, ethnicity,
 age, religion, nationality, or other such characteristics.
 
-Everyone is expected to follow the [Scala Code of Conduct] when discussing the
+Everyone expected to follow the [Scala Code of Conduct], when discussing the
```

Comment:
I think "Everyone is expected" is more appropriate in this context.
[Example of "Everyone is expected"](https://ludwig.guru/s/everybody+is+expected)

Reply:
👍